### PR TITLE
[ENG-2691] No-Project email fixes

### PR DESCRIPTION
**Pull request by jwalz**
## Purpose
Make sure initiators of a "No-Project" Draft Registration receive an email with the link to the registration.

Draft Registrations that are not associated with a Project currently have no means of discovery beyond the email link sent to contributors. Historically, contributors

## Changes
* Call the `contributor_added` signal in `DraftRegistration.create_from_node` when creating a DraftRegistration without a backing project.
* Update the `send_mail` call when notifying new contributors to pass all args as kwargs to improve hygiene
* Update the `URL_TEMPLATE` for DraftRegistrations to match the correct, updated format
* Minor fix to email text for new Draft Registration contributors
* Tests to confirm new behavior

## QA Notes
New tests verify that email is sent to the initiator iff the draft node is created without a `branched_from` node.

QA/Product should verify email text

What are the areas of risk?

Any concerns/considerations/questions that development raised?

## Ticket
https://openscience.atlassian.net/browse/ENG-2691

### CODE REVIEW

**Review by UdayVarkhedkar — CHANGES_REQUESTED** [review]
Looks good outside of what should be that quick change to the `url` property on the draftregistration class!

File: osf/models/registrations.py [hunk 1]
```diff
@@ -759,7 +760,7 @@ class DraftRegistration(ObjectIDMixin, RegistrationResponseMixin, DirtyFieldsMix
         'node_license',
     ]
 
-    URL_TEMPLATE = settings.DOMAIN + 'project/{node_id}/drafts/{draft_id}'
+    URL_TEMPLATE = settings.DOMAIN + 'registries/drafts/{draft_id}'
```
[the remark this answers is not in the document]
Comment (jwalz) [reply osf/models/registrations.py:763]: Good catch! I'm excited to see how many tests this broken format call breaks.

**Review by UdayVarkhedkar — APPROVED** [review]

File: osf/models/registrations.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (corbinSanders) [reply osf/models/registrations.py:763]: How many tests did it end up breaking?

[the remark this answers is not in the document]
Comment (jwalz) [reply osf/models/registrations.py:763]: none due to the .format call, somehow; one due to explicitly testing for the old url format

**Pull request merged**